### PR TITLE
Create fixtures for recommendation requests

### LINKED ISSUE #55: Create recommendation requests fixtures
- Create recommendation request fixtures based on the entity that @divyagsubramonian is creating.

### PR DESCRIPTION
**Pull request by Ramon-W**
#### Description
- Added `oneRecommendationRequest` and `threeRecommendationRequests`
- These will be used to mock fetched data for frontend tests get recommendation requests.

#### Testing
- No tests needed

Closes #55

### CODE REVIEW

**Review by kanav9063 — DISMISSED** [review]
both oneRecommendationRequest and threeRecommendationRequests in the fixtures file looks correct !

File: frontend/src/fixtures/recommendationRequestsFixtures.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+const recommendationRequestsFixtures = {
+  oneRecommendationRequest: [
+    {
+      id: 1,
+      requesterEmail: "ramonwang@ucsb.edu",
+      requesterName: "Ramon Wang",
+      professorEmail: "pconrad@ucsb.edu",
+      professorName: "Phil Conrad",
+      requestType: "CS Department BS/MS program",
```
Comment (pconrad) [inline frontend/src/fixtures/recommendationRequestsFixtures.js:9]: Ultimately, these might be better as foreign keys into the users table (for requester and professor) and for requestType.  But for now, this is fine.

**Review by pconrad — DISMISSED** [review]
I have some feedback below, but this is ok for now.  LGTM

File: frontend/src/fixtures/recommendationRequestsFixtures.js (hunk 1 above)
Comment (Ramon-W) [reply frontend/src/fixtures/recommendationRequestsFixtures.js]: Okay 👍  we'll use foreign keys once those are set up.

**Review by kanav9063 — APPROVED** [review]
the name change from frontend/src/fixtures/recommendationRequestsFixtures.js to frontend/src/fixtures/recommendationRequestFixtures.js was correctly changed

**Review by pconrad — APPROVED** [review]
LGTM... though some later adjustments might be needed.

**Pull request merged**